docs/source/getting_started/changelog.rst

```diff
@@ -3,6 +3,91 @@
 JupyterLab Changelog
 ====================
 
+TODO after release: convert to link:
+`v2.0 https://github.com/jupyterlab/jupyterlab/releases/tag/v2.0`__
+---------------------------------------------------------------------------
+
+2020/01/03
+^^^^^^^^^^
+
+See the `JupyterLab
+2.0 <https://github.com/jupyterlab/jupyterlab/milestone/2.0?closed=1>`__
+milestone on GitHub for the full list of pull requests and issues closed.
+
+User-facing changes
+^^^^^^^^^^^^^^^^^^^
+* New celltags UI (`#7407 <https://github.com/jupyterlab/jupyterlab/pull/7407>`__)
+* Support for output search in notebooks (`#7258 <https://github.com/jupyterlab/jupyterlab/pull/7258>`__)
+* Add commands to move to next/prev tab bar in shell (`#7673 <https://github.com/jupyterlab/jupyterlab/pull/7673>`__)
+* Simplify user experience when selecting 'no kernel' for notebook (`#7647 <https://github.com/jupyterlab/jupyterlab/pull/7647>`__)
+* Drops support for Vega 4 and Vega-lite 2 rendereding (`#7650 <https://github.com/jupyterlab/jupyterlab/pull/7650>`__, `#7523 <https://github.com/jupyterlab/jupyterlab/issues/7523>`__)
+* Extra flags for ``jupyter lab clean``: --extensions, --settings, --static, --all (`#7583 <https://github.com/jupyterlab/jupyterlab/pull/7583>`__)
+* Global track timing setting for notebooks (`#7578 <https://github.com/jupyterlab/jupyterlab/pull/7578>`__)
+* Better command palette icon (`#7577 <https://github.com/jupyterlab/jupyterlab/pull/7577>`__, `#7038 <https://github.com/jupyterlab/jupyterlab/issues/7038>`__)
+* Added a shortcut to go to the last run or current running cell (`#7551 <https://github.com/jupyterlab/jupyterlab/pull/7551>`__)
+* File info on file browser hover (`#7485 <https://github.com/jupyterlab/jupyterlab/pull/7485>`__, `#7352 <https://github.com/jupyterlab/jupyterlab/issues/7352>`__)
+* Add new file and new markdown file to file browser context menu (`#7483 <https://github.com/jupyterlab/jupyterlab/pull/7483>`__, `#4280 <https://github.com/jupyterlab/jupyterlab/issues/4280>`__)
+* In extension manager, show "enable" button only for installed extensions (`#7482 <https://github.com/jupyterlab/jupyterlab/pull/7482>`__)
+* Add download command to file menu (`#7480 <https://github.com/jupyterlab/jupyterlab/pull/7480>`__)
+* Added the selection of cells till top and bottom (`#7177 <https://github.com/jupyterlab/jupyterlab/pull/7177>`__)
+
+For developers
+^^^^^^^^^^^^^^
+* Switch from ``@phosphor` to ``@lumino`` dependencies. (`#7582 <https://github.com/jupyterlab/jupyterlab/pull/7582>`__, `#7534 <https://github.com/jupyterlab/jupyterlab/issues/7534>`__)
+* The ``settingsregistry`` and ``statedb`` packages are broken out from coreutils (`#7681 <https://github.com/jupyterlab/jupyterlab/pull/7681>`__, `#7615 <https://github.com/jupyterlab/jupyterlab/issues/7615>`__)
```

Comment:
```suggestion
* The ``settingsregistry`` and ``statedb`` packages are factored out from coreutils (`#7681 <https://github.com/jupyterlab/jupyterlab/pull/7681>`__, `#7615 <https://github.com/jupyterlab/jupyterlab/issues/7615>`__)
```

(want to avoid the word "broken" :)